src/utils/geminiai.ts

```diff
@@ -0,0 +1,7 @@
+import { GoogleGenerativeAI } from "@google/generative-ai";
```

Comment:
these act more like services, Create a folder for services which will have firebase, gemini and other configurations setup

Reply:
i create another branch for folder structure